src/AppBundle/Repository/NearbyTrait.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Geocoder\Coordinates;
+use Doctrine\ORM\QueryBuilder;
+
+trait NearbyTrait
+{
+    /**
+     * @return string
+     */
+    public function getNearbyExpression(): string
+    {
+        return '(6371 * acos(cos(radians(:latitude)) * cos(radians(n.postAddress.latitude))
+            * cos(radians(n.postAddress.longitude) - radians(:longitude)) + sin(radians(:latitude)) *
+            sin(radians(n.postAddress.latitude))))';
+    }
+
+    /**
+     * Calculates the distance (in Km) between the committee and the provided geographical
```

Comment:
I suppose you use this also for events, so I wouldn't put the name of the entity in the comment.